tests/unit/effects/core.js

```diff
@@ -258,6 +258,29 @@ test( "createPlaceholder: preserves layout affecting properties", function() {
 	deepEqual( before.outerHeight, placeholder.outerHeight( true ), "height preserved" );
 });
 
+module( "transfer" );
+
+asyncTest( "transfer() without callback", function() {
+	expect( 0 );
+	$( "#elem" ).transfer( {
+		to: ".animateClass",
+		duration: 1
+	} );
+	setTimeout( function() {
+		start();
+	}, 25 );
+} );
+
+asyncTest( "transfer() with callback", function() {
+	expect( 0 );
+	$( "#elem" ).transfer( {
+		to: ".animateClass",
+		duration: 1
+	}, function() {
+		start();
```

Comment:
Seems like you could add an `ok( true, "callback invoked" )` to at least have an assertion here.